test/e2e-tests.sh

```diff
@@ -70,8 +70,8 @@ install_knative_serving || fail_test "Knative Serving installation failed"
 publish_test_images || fail_test "one or more test images weren't published"
 
 # Run the tests
-
 header "Running tests"
+kubectl create namespace serving-tests
```

Comment:
Moving this here will break the other tests (performance, upgrade/downgrade, etc). All E2E tests expect the images to be pushed to the `serving-tests` namespace.